analyses/focal-cn-file-preparation/01-add-ploidy-cnvkit.Rmd

```diff
@@ -26,10 +26,63 @@ histologies_file <- file.path("..", "..", "data", "histologies.tsv")
 histologies_df <- readr::read_tsv(histologies_file, guess_max = 100000)
 ```
 
+# Edits for https://github.com/PediatricOpenTargets/ticket-tracker/issues/177
+```{r}
+# Filter to WXS tumor with no germline sex estimate
+no_estimate <- histologies_df %>% 
+  filter(sample_type == "Tumor") %>% 
+  filter(experimental_strategy == "WXS") %>%  
+  filter(is.na(germline_sex_estimate)) %>%
+  select(-germline_sex_estimate)
+
+# Extract participant ID with no germline sex estimate
+no_estimate_participants <- no_estimate %>% 
+  pull(cohort_participant_id) %>% 
+  unique()
```

Comment:
```suggestion
no_estimate_participants <- no_estimate %>% 
  pull(Kids_First_Participant_ID) %>% 
  unique()
```

I would use the `Kids_First_Participant_ID` here because that is our main ID from which we are working - theoretically these are 1:1 with `cohort_participant_id`, but the `cohort_participant_id` is metadata that "comes along" with the samples and we like to stick to KF ids.

Reply:
Done